### DISCUSSION TITLE
Is it expected behaviour that "What's up?" is classified as TSWANA?

### DISCUSSION THREAD

**Discussion by Paethon** [pemistahl/lingua-py#59] [General]
Hi

I guess the title says it all. When I classify the sentence "What's up?" without restricting the languages to consider, the sentence is classified as being Tswana instead of English. I am of course aware that using statistical models etc. and very short texts, something like this can happen.

I just want to check if this is expected behaviour or a bug after all.

Cheers, Sebastian

**Comment by pemistahl** [comment]
Hi Sebastian, thanks for your question.

No, this is not a bug. For the text "What's up?", the sum of the ngram probabilities for Tswana is simply greater than the sum of the ngram probabilities for English. If you take the text "What is up?" instead, the detector will return English.

**Reply by Paethon** [reply]
Thanks. That is what I expected. Just wanted to make sure 😁
